src/main/java/com/epam/ta/reportportal/ws/model/FinishExecutionRQ.java

```diff
@@ -59,6 +58,8 @@
 	@JsonProperty
 	private Set<ItemAttributesRQ> attributes;
 
+	private String baseURL;
```

Comment:
why do we need it?

Reply:
agree
it shouldn't be here
left a comment in api about it

Reply:
agree
it shouldn't be here
left a comment in api about it